tools/quic_trace_to_time_sequence_gnuplot.cc

```diff
@@ -20,14 +20,19 @@
 
 #include "absl/container/flat_hash_map.h"
 #include "absl/container/flat_hash_set.h"
-#include "gflags/gflags.h"
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
 #include "lib/quic_trace.pb.h"
 
-DEFINE_string(sequence, "Which time sequence to plot: send, ack or loss.", "");
-DEFINE_bool(filter_old_acks,
-            true,
-            "Do not show an ack for a packet if it was already previously "
-            "acknowledged");
+ABSL_FLAG(std::string,
+          sequence,
+          "Which time sequence to plot: send, ack or loss.",
```

Comment:
I think the two strings are reversed here: the third parameter is the default value and the fourth is the help string

Reply:
Fixed.